docs/multimodal_assistant_mask.md

```
@@ -0,0 +1,84 @@
+# Multimodal assistant-only loss masking
+
+### Correct placement
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)